packages/webcert/src/components/question/QuestionForm.tsx

```diff
@@ -41,10 +41,10 @@ const QuestionForm: React.FC<Props> = ({ questionDraft }) => {
   const isSaved = useSelector(isQuestionDraftSaved)
   const isMissingType = useSelector(isQuestionMissingType)
   const isMissingMessage = useSelector(isQuestionMissingMessage)
-  const isSending = useSelector(isSendingQuestion)
   const showValidationMessages = useSelector(isDisplayValidationMessages)
   const [message, setMessage] = useState(questionDraft.message)
   const subjects: QuestionType[] = Object.values(QuestionType)
+  const isFunctionBlocked = useSelector(isQuestionFunctionDisabled)
```

Comment:
For consistency it should be called isFunctionDisabled instead of isFunctionBlocked.